apis/vpcresources/v1alpha1/cninode_types.go

```diff
@@ -25,15 +25,22 @@ const (
 	CustomNetworking      FeatureName = "CustomNetworking"
 )
 
+// Feature is a type of feature being supported by VPC resource controller and other AWS Services
+type Feature struct {
+	Name  FeatureName `json:"name,omitempty"`
+	Value string      `json:"value,omitempty"`
```

Comment:
What are these values?

Reply:
This is for custom networking CRD eniconfig's name. I was planning to call CNI's API to get the value instead of asking CNI to add into the CRD. But I realized the API won't work since it depends on aws-node local ENVs. Checking ENVs in pods won't be 100% reliable since users can set them up by different ways.